### PR TITLE
minimize ubuntu deps. needed to run build openpilot

### PR DESCRIPTION
**Pull request by BBBmau**
Continuation of #32475

Goal is to get all CI tests passing with the least amount of dependencies needed

### CODE REVIEW

File: tools/install_ubuntu_dependencies.sh [hunk 1]
```diff
@@ -63,25 +54,35 @@ function install_ubuntu_common_requirements() {
...
     casync \
     cmake \
     make \
-    libdw1
+    clinfo \
+    libqt5sql5-sqlite \
+    libqt5charts5-dev \
+    libreadline-dev \
+    libdw1 \
+    libbz2-dev \
```
Comment (adeebshihadeh) [inline tools/install_ubuntu_dependencies.sh:75]: i think we can just delete this. it was for when loggerd would write out compressed logs directly, but the uploader does compression now

**Comment by BBBmau** [comment]
Nice, it's builds openpilot just fine with the latest commit.

Next would be figuring out which packages are now needed for passing tests:

Required packages JUST to build openpilot:
```

**Comment by adeebshihadeh** [comment]
So many packages... we've gotta get that down. This is a great start though!

**Comment by BBBmau** [comment]
> So many packages... we've gotta get that down. This is a great start though!

I think this is a pretty good cut down, these are the packages that we really don't need. I really wasn't expecting it to be this much 😄 
```
    casync \
    cmake \
    make \
    clinfo \
    libqt5sql5-sqlite \
    libreadline-dev \
    libdw1 \
    autoconf \
    libtool \
    bzip2 \
    libarchive-dev \
    libncursesw5-dev \
    libportaudio2 \
    locales
 ```
 
 @adeebshihadeh this should be ready to merge

**Pull request merged**